StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/WalletHeaderView.swift

```diff
@@ -182,3 +182,17 @@ extension PaymentSheetViewController {
         }
     }
 }
+
+// MARK: - EventHandler
+extension PaymentSheetViewController.WalletHeaderView: EventHandler {
+    func handleEvent(_ event: STPEvent) {
+        switch event {
+        case .shouldEnableUserInteraction:
+            alpha = 1
+        case .shouldDisableUserInteraction:
+            alpha = 0.8
```

Comment:
How'd we land on 0.6 for the rest of the UI for 0.8 here?

Reply:
I see some 0.5's too

Reply:
I just eyeballed it. Will get Hannah to review next week.